examples/nft_collection/01_mint_collection_nft.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example mint_collection_nft --release
+// In this example we will mint the 150 nfts with issuer feature.
+// Rename `.env.example` to `.env` and run 01_create_wallet.rs before
+
+use std::{env, str::FromStr};
+
+use dotenv::dotenv;
+use iota_client::block::{
+    address::{Address, NftAddress},
+    output::NftId,
+};
+use iota_wallet::{account_manager::AccountManager, NftOptions, Result};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
+    dotenv().ok();
+
+    let nft_collection_size = 15;
+    // Set this to the NFT id from the mint_issuer_nft example
+    let issuer_nft_id = NftId::from_str("0x13c490ac052e575cffd40e170c2d46c6029b8b68cdf0e899b34cde93d2a7b28a")?;
```

Comment:
How do they get it from min_issuer_nft?

Reply:
They have to check the balance for the new NFT ID, get it form the explorer or whatever :trollface:
Should I add some code to print the NFT ID?

Reply:
That would be very nice 🙏🏻